kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1239,6 +1239,21 @@ object KyuubiConf {
       .booleanConf
       .createWithDefault(false)
 
+  val KUBERNETES_APPLICATION_STATE_FROM_CONTAINER: ConfigEntry[Boolean] =
+    buildConf("kyuubi.kubernetes.application.state.fromContainer")
```

Comment:
actually, there are other potential ways for app state judgement, e.g. service, or CRD object, so I would suggest using an emum instead of bool for it.
```
kyuubi.kubernetes.application.state.from=[pod|contianer]
```